.mailmap

```diff
@@ -59,6 +59,8 @@ Matt Keller <matt@zulip.com>
 Matt Keller <matt@zulip.com> <m@cognusion.com>
 m-e-l-u-h-a-n <purushottam.tiwari.cd.cse19@itbhu.ac.in>
 Noble Mittal <noblemittal@outlook.com> <62551163+beingnoble03@users.noreply.github.com>
+Palash Baderia <palash.baderia@outlook.com> <palash.baderia@outlook.com>
+Palash Baderia <66828942+palashb01@users.noreply.github.com> <palash.baderia@outlook.com>
```

Comment:
This produces this output from `git shortlog -es | sort -k2`, is that your intent?
```
    40  Palash Baderia <66828942+palashb01@users.noreply.github.com>
     1  PALASH BADERIA <66828942+palashb01@users.noreply.github.com>
```

Reply:
@timabbott , I have just fixed it. Thank you for the ping. Initially, I thought we needed to have the secondary email as the first parameter and then the primary email we want to display as the second parameter. However, I have now fixed it.

```console

41	Palash Baderia <palash.baderia@outlook.com>

```